lib/flipper/expression/constant.rb

```diff
@@ -0,0 +1,25 @@
+module Flipper
+  class Expression
+    # Public: A constant value like a "string", Number (1, 3.5), Boolean (true, false).
+    #
+    # Implements the same interface as Expression
+    class Constant
+      include Expression::Builder
+
+      attr_reader :value
+
+      def initialize(value)
+        @value = value
+      end
+
+      def evaluate(_ = nil)
```

Comment:
Super minor: this could just be `-` right? `-` is usually to denote unused, so assigning something that is unused seems pointless.

Reply:
Ah. Didn’t think of that. Ignore me.